react/src/lib/EventOverlay/index.js

```diff
@@ -686,15 +686,14 @@ class EventOverlay extends React.Component {
         break;
       case 'left':
         if (!scrollParentDims.left && !transformParentDims) {
-          if (
-            arrowWidth + offsetWidth + elementDims.width + anchorPosition.left >
-            anchorPosition.left
-          ) {
+          targetNode.style.right = `${documentRight -
+            anchorPosition.left +
+            arrowWidth +
+            offsetWidth}px`;
+          if ( elementDims.left - arrowWidth < 0 ) {
```

Comment:
The minimum left placement possible is arrowWidth. This is due to a negative arrowWidth margin applied in the common css for left hand placed overlays.